### DISCUSSION TITLE
Remind me to lock my car

### DISCUSSION THREAD

**Discussion by gerard33** [bimmerconnected/bimmer_connected#253] [Show and tell]
See [this topic](https://community.home-assistant.io/t/bmw-connected-automation-remind-me-to-lock-my-car/245802?u=gerard33) on the HA forum by simbieski where he describes to send reminders when the car is not locked, including actionable notifications for iOS users 👍

**Comment by gurmukhp** [comment]
This is essential, coming from a Tesla I often forget to lock the car and so would love HA to do it for me, any chance we can do this?

**Reply by rikroe** [reply]
Your car will update its position in HA everytime it is shut down/stopped. I guess this should also apply if it is not locked.

Then you could build a HA automation on location state change if car is unlocked.
